src/Illuminate/Testing/Comparators/ModelComparator.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+namespace Illuminate\Testing\Comparators;
+
+use Illuminate\Database\Eloquent\Model;
+use SebastianBergmann\Comparator\Comparator;
+use SebastianBergmann\Comparator\ComparisonFailure;
+
+class ModelComparator extends Comparator
+{
+    public function accepts($expected, $actual): bool
+    {
+        return $expected instanceof Model && $actual instanceof Model;
+    }
+
+    /**
+     * @param Model $expected
+     * @param Model $actual
+     */
```

Comment:
The docblocks on this and the above method don't match the rest of the framework. Please use FQN, double spaces after/before param types, add a return type and a short description.

Reply:
@olivernybroe the second docblock is still missing the two spaces before and after the types.

Reply:
Right, sorry about that 😆